src/components/camera-tool.js

```diff
@@ -623,6 +623,14 @@ AFRAME.registerComponent("camera-tool", {
           }
         }
 
+        const bubbleSystem = this.el.sceneEl.systems["personal-space-bubble"];
+
+        if (bubbleSystem) {
+          for (const invader of bubbleSystem.invaders) {
+            invader.disable();
+          }
```

Comment:
Just wanted to note that you're using a `for...of` loop in a `tock` here. Perhaps it's not a concern if the short-lived allocations we've observed can indeed be ignored.

Reply:
good point i'll refactor